examples/prompts/llm_monitoring/openai_client_azure.ipynb

```diff
@@ -0,0 +1,91 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "385aadd9-e901-4b43-97ca-cc10cf920e9c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# adpated from Azure docs here: https://learn.microsoft.com/en-us/azure/ai-services/openai/quickstart?pivots=programming-language-python\n",
+    "\n",
+    "# Set the openai settings on the base openai object\n",
+    "import openai\n",
```

Comment:
this extra import may look weird but it is VERY important. Since weave shadow the openai object, you must first import open and set the settings on the original package. Then you can import weave openai (i choose to rename it so it is clear). We should fix this so users don't run into this issue.